### ISSUE TITLE
Create a Rules section

### ISSUE THREAD

**Issue by TheFleetCommander** [hellopaint-io/hellopaint#28]
Current: HelloPaint does not have any rules publicly available on-site. 

Expected: There should be a dedicated section for rules on HP like there is on Malmal

**Comment by lucasmerlin** [comment]
I have a PR open where I add a docs section to HelloPaint, I plan on moving the docs to this repo so mods/anyone can edit them. 
I think we can just have a rules page in the docs and link to that in relevant places?

**Comment by Krowphay** [comment]
If we can include a button on the home page that clearly links to the rules in the process, I think that would work just fine personally

**Comment by TheFleetCommander** [comment]
> I have a PR open where I add a docs section to HelloPaint, I plan on moving the docs to this repo so mods/anyone can edit them. I think we could just have a rules page in the docs and link to that in relevant places?

I agree with Krow, as long as it's easily accessible and obvious where people can view rules, this should work just fine
